sdks/python/apache_beam/runners/dataflow/internal/dependency.py

```diff
@@ -77,7 +77,7 @@
 
 # Update this version to the next version whenever there is a change that will
 # require changes to legacy Dataflow worker execution environment.
-BEAM_CONTAINER_VERSION = 'beam-master-20171214'
+BEAM_CONTAINER_VERSION = 'beam-master-20180228'
 # Update this version to the next version whenever there is a change that
 # requires changes to SDK harness container or SDK harness launcher.
 BEAM_FNAPI_CONTAINER_VERSION = 'beam-master-20171222'
```

Comment:
Why not update this as well?

Reply:
We don't have to unless there were changes to python-fnapi container image. Looks like this time there were changes container image, so it's a good idea to update.